mali_counter.h

```diff
@@ -58,16 +58,21 @@ class MaliCounter : public Instrument
 	void            sample_counters();
 	void            wait_next_event();
 	const uint32_t *get_counters() const;
-	const uint32_t *get_counters(mali_userspace::MaliCounterBlockName block, int core = -1) const;
+	const uint32_t *get_counters(mali_userspace::MaliCounterBlockName block, int index = -1) const;
 	int             find_counter_index_by_name(mali_userspace::MaliCounterBlockName block, const char *name);
 
+	const std::vector<const std::pair<const char *, const char *>> _jm_counter_names{
+	    {"GPU_ACTIVE", "cycles"}, {"JS0_JOBS", "jobs"}, {"JS1_JOBS", "jobs"}};
+	const std::vector<const std::pair<const char *, const char *>> _mmu_counter_names{
+	    {"L2_READ_LOOKUP", "cache lookups"}, {"L2_EXT_READ", "transactions"}, {"L2_EXT_AR_STALL", "stall cycles"}, {"L2_WRITE_LOOKUP", "cache lookups"}, {"L2_EXT_WRITE", "transactions"}, {"L2_EXT_W_STALL", "stall cycles"}, {"L2_EXT_READ_BEATS", "bus cycles"}, {"L2_EXT_WRITE_BEATS", "bus cycles"}};
```

Comment:
can we change the formatting to make this list more readable and easier to expand ?

Reply:
Done in https://github.com/ARM-software/HWCPipe/pull/7/commits/12c14b6ad8acb9c2782d2e4b3b82711bb337b8ec